### PR TITLE
Fix sitemap route

### LINKED ISSUE #37: How to call the sitemap.xml
I used the Docs to install and configure it. My very simple Question is how to view it, the Plugin/config/routes.php let me guess it is Domain/sitemap oder /sitemap.xml , but this just shows:

```
Documentation API
Error: SitemapController could not be found.

Error: Create the class SitemapController below in file: src/Controller/SitemapController.php
```

### PR DESCRIPTION
**Pull request by elboletaire**
Fixes #37

I've tried to add a test to check it, but it crashes due to the `routes.php` file not being loaded properly. Sorry for that.

The problem with the routes was that you were using the `dashedRoutes` fallback, which was generating urls of this type (extracted from RoutesShell):

    | sitemap._controller:index               | /sitemap/:controller                                       | {"action":"index","plugin":"Sitemap"}                                                              |
    | sitemap._controller:_action             | /sitemap/:controller/:action/*                             | {"action":"index","plugin":"Sitemap"}

With my changes, the expected route now is `/sitemap(.xml)?`:

    | sitemap.sitemaps:index                  | /sitemap                                                   | {"controller":"Sitemaps","plugin":"Sitemap","action":"index"}                                      |

### CODE REVIEW

**Comment by elboletaire** [comment]
I've been able to fix the tests, after refactoring a bit. Let me know if it looks ok for you.

**Comment by justinyost** [comment]
FYI, I just merged a PR that fixes the CS and updates the tests a tad, please rebase this and I can take a look at it. Thanks.

**Comment by elboletaire** [comment]
Done

**Comment by justinyost** [comment]
HI @elboletaire can you add a docblock to the `testIndexAccess` method.

Something like:

```
/**
 * Test the index method that the correct route is loaded and works. Essentially
 * this test validates the routes.php file as being correct and working.
 *
 * @return void
 */
```

Thanks.

File: tests/TestCase/Controller/SitemapsControllerTest.php [hunk 1]
```diff
@@ -103,4 +103,10 @@ public function testIndexWithModels() {
 
 		$Controller->index();
 	}
+
+	public function testIndexAccess() {
```
Comment (justinyost) [inline tests/TestCase/Controller/SitemapsControllerTest.php]: Docblock here please something along the lines of:

/**
 * Test the index method that the correct route is loaded and works. Essentially
 * this test validates the routes.php file as being correct and working.
 *
 * @return void
 */

**Pull request merged**